mne/realtime/mock_lsl_stream.py

```diff
@@ -0,0 +1,98 @@
+# Authors: Teon Brooks <teon.brooks@gmail.com>
+#
+# License: BSD (3-clause)
+import time
+from multiprocessing import Process
+
+from numpy.random import rand
+import numpy as np
+
+from ..utils import _check_pylsl_installed
+
+
+class MockLSLStream:
```

Comment:
should inherit from object

Reply:
you don't need to inherit from Object in Python3. and since we're no longer supporting Py2, this is now valid :)